examples/Test/Orders/003_AL_ORM_O01_NBS_Fully_Populated_0_initial_message.hl7

```diff
@@ -1,4 +1,4 @@
-MSH|^~\&|BaptistOracle^2.16.840.1.114222.4.1.000000^ISO|BaptistEast^2.16.840.1.114222.4.1.000001^ISO|ALlabNatus^2.16.840.1.114222.4.1.181960.2^ISO|ALlab^simulated-lab-id^ISO|20240224134009||ORM^O01^ORM_O01|Q1960841872T2476960690||2.5.1||||||8859/1
+MSH|^~\&|BaptistOracle^2.16.840.1.114222.4.1.000000^ISO|BaptistEast^2.16.840.1.114222.4.1.000001^ISO|ALlabNatus^2.16.840.1.114222.4.1.181960.2^ISO|ALlab^simulated-lab-id^ISO|20240224134009||ORM^O01^ORM_O01|Q1960841872T2476960690|D|2.5.1||||||8859/1
```

Comment:
should the corresponding final message for these also be updated?

Reply:
See my other response